packages/wonder-blocks-form/src/components/field-heading.js

```diff
@@ -28,6 +28,11 @@ type Props = {|
      */
     description?: string | React.Element<Typography>,
 
+    /**
+     * Whether this field is required to to continue.
```

Comment:
Typo: 
```suggestion
     * Whether this field is required to continue.
```

What do you think about framing this in a way where you describe the validation intention?

Something like `Whether this field is required and validated`